### ISSUE TITLE
Search in module name and optional description

### ISSUE THREAD

**Issue by spekulatius** [silverstripe/addons.silverstripe.org#118]
tl;dr: Search in module name only as a feature would be great.

Some search results aren't optimal IMHO as by default the readme-files of the module as well as the title are searched. For example this http://addons.silverstripe.org/add-ons?search=composer&type=&sort=name returns a long list because it obviously is a very common term in the readme-files, but not in the title. Modules related to "composer" like my module https://github.com/spekulatius/silverstripe-composer-security-checker aren't showing up really. Its not like I care about my module. I care about the/our/my user experience. 

If we make the search scope optional for the readme-files the user experience could be better IMHO. Is there interest in this feature? If so, I would be interested to discuss here how exactly the user experience could be improved based on this (e.g. pre-selected by default? As a fall back in case there aren't any results? As a suggestion in case there aren't any results?). 

Thanks for reading all of this :)

**Comment by camfindlay** [comment]
Hey @spekulatius that's for chiming in on the addons site. It's one of those sites we'd like to give some love at some point, in the meantime there are some community members interested in seeing what can be done. Perhaps a small community workgroup could be formed around this site to get some movement on it faster? @SpiritLevel might be work chatting to. As far as search goes... there has been a small amount of discussion looking at perhaps moving to Solr search for this site, it would be good to address things like weighting of fields, boosting and other useful search mechanisms.

**Comment by robbieaverill** [comment]
Search has been improved slightly in #200, feel free to reopen if this is still an issue outside of that
